src/openms/source/APPLICATIONS/ConsoleUtils.cpp

```diff
@@ -121,6 +120,7 @@ namespace OpenMS
         {
           LOG_DEBUG << "output shaping: stty size command failed." << std::endl;
         }
+        pclose(fp);
```

Comment:
not so sure about that
`If the argument stream to pclose() is not a pointer to a stream created by popen(), the result of pclose() is undefined.` 
which to me seems exactly the case when you give fp which is equal to `NULL` to the fxn. We dont want undefined because that could mean the code takes your credit card and plays online poker with it, or something like that